CONFIGDOC.md

```diff
@@ -20,14 +22,13 @@ When you define an automated pipeline of tasks in Jenkins&trade;, whether in the
 You can use the web UI provided by Jenkins to configure the plugin in freestyle and multi-configuration projects. To run MATLAB or Simulink in a Pipeline project, see [Set Up Pipeline Project](#set-up-pipeline-project).
 
 ### Use MATLAB in Build
-Once you install the plugin, **Use MATLAB version** appears in the **Build Environment** section of the project configuration window.
+Once you install the plugin, **Use MATLAB version** appears in the **Build Environment** section of the project configuration window. Select **Use MATLAB version** to specify the MATLAB version you want to use in the build. You can select one of the MATLAB versions that have been registered as a Jenkins tool, or you can select **Custom** if you want to specify a different version. For more information about registering MATLAB as a tool, see [Register MATLAB as Jenkins Tool](#register-matlab-as-jenkins-tool).
 
+In this example, the list includes two registered tools as well as the option for specifying a custom installation. If you select **Custom**, then you must enter the full path to your desired MATLAB root folder in the **MATLAB root** box.

```

Comment:
Should  we add a screenshot for Custom location here as well?

Reply:
I suggest we do not add a screenshot for these reasons:

1) The text clearly describes what will happen in the UI.
2) We already have two screenshots on the page that display the MATLAB root box.
3) While screenshots are nice, they are difficult to maintain.

I just updated the text as follows to better describe the MATLAB root box. However, if you feel strongly about having a screenshot, please let me know.

"If you select Custom, a MATLAB root box appears in the UI. You must enter the full path to your desired MATLAB root folder in this box."

Reply:
Sounds good.